template/src/lib/ui/stories/common/Tooltip.stories.js

```diff
@@ -119,13 +128,18 @@ const DataComponent = () => {
 };
 
 const TooltipDataTemplate = () => {
+  const DataTooltip = withStyles((theme) => ({
+    tooltip: {
+      padding: theme.spacing(1, 1.5),
+    },
+  }))(Tooltip);
```

Comment:
The storybook should show what do you get with no custom styles. Maybe a bit of styling is ok if it is justified; but in this case I see maybe too mucho.

Reply:
it's only for the specific case. Is not possible to apply two styles for the same tooltip. The only solution would be create a DataTooltip, but I think that it's not necessary. So, I think that we can remove this example or create the DataTooltip.
What do you think?

Reply:
To fix

Reply:
Better define custom padding on the `TooltipContent` component